File: Data Structures & Algorithms/concatenation-of-array/submission-0.cpp
class Solution {
public:
    vector<int> getConcatenation(vector<int>& nums) {
        std::vector<int> ans = {};
        int current_size = nums.size();
        ans.resize(current_size * 2);
        for (int idx = 0; idx < current_size; idx++) {
            ans[idx] = nums[idx];
            ans[idx + current_size] = nums[idx];
        }
        return ans;
    }
};

File: Data Structures & Algorithms/two-integer-sum/submission-0.rs
impl Solution {
    pub fn two_sum(nums: Vec<i32>, target: i32) -> Vec<i32> {
    for (idx, num) in nums.iter().enumerate() {
        for (idy, val) in nums.iter().enumerate().skip(idx + 1) {
            if val + num == target {
                return vec![idx as i32, idy as i32];
            }
        }
    }
    vec![]

    }
}
